modules/installation-gcp-enabling-confidential-vms.adoc

```diff
@@ -20,7 +20,7 @@ include::snippets/technology-preview.adoc[]
 
 [IMPORTANT]
 ====
-Due to a known issue, you cannot use persistent volume storage on a cluster with Confidential VMs. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-7582[OCPBUGS-7582].
+Due to a known issue in {product-title} 4.13.3 and earlier versions, you cannot use persistent volume storage on a cluster with Confidential VMs. This issue was resolved in {product-title} 4.13.4. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-7582[OCPBUGS-7582].
```

Comment:
In previous doc, there is only one bug OCPBUGS-7582 to track this and it is for 4.14, and [OCPBUGS-11768](https://issues.redhat.com/browse/OCPBUGS-11768) is for 4.13 and we have verify info there, so I recommend to use OCPBUGS-11768 here to avoid confusing when checking that bug, unless you have any other concern like readers would like to keep the original ID.

Reply:
Thanks, @duanwei33 . I updated all the links to now target OCPBUGS-11768.